src/index.ts

```diff
@@ -168,6 +168,9 @@ export class Connection {
     }
 
     if (config.url) {
+      if (!config.url.startsWith('http')) {
```

Comment:
By design, it's intended to ignore the scheme since it's common that a `DATABASE_URL` env var or something is shared as `DATABASE_URL=mysql://` and we are just making that format work for us. We never speak the mysql protocol in this library, nor does `mysql://` in a url really mean anything other than a hint.

So I'd strongly disagree with this check and would break expectations.

Reply:
Okay I'm fine with that, I just wanted to give it a try to provide some better dx by not stumbling over this issue